pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -222,6 +230,14 @@ func (c *Cluster) makeDeployment(nodeName string, selection rookalpha.Selection,
 			args = append(args, "--default-log-to-file", "false")
 		}
 
+		// OSD fails to find the right IP to bind to when running on SDN
```

Comment:
This is the same code as the previous scenario? Could you factor the common code out to a helper method to avoid the duplication?

Reply:
You mean it's the same line block as 175?
That's how the spec is done for most of the flag currently but I can look at improving this.

Reply:
Done!